### ISSUE TITLE
Resubmit to CRAN?

### ISSUE THREAD

**Issue by briatte** [expersso/gesis#14]
Hello,

I guess it's not a priority, given how easy it is to install packages from GitHub, but I was saddened to see that CRAN had removed your excellent package. Resubmit one day, perhaps?

(I confess I do not know if packages can be resubmitted after removal).

All the best,

François

**Comment by fsolt** [comment]
Seconded.  I'm willing to assist.

**Comment by fsolt** [comment]
Wow.  CRAN archived it five days after a new version?  'Cause Solaris still wasn't happy?  🤬

**Comment by expersso** [comment]
Yeah, I got an email that it would get archived unless I fixed a unit test that was failing on Solaris. I thought I’d fixed it, only to learn that it had been archived when it was mentioned as an issue here. 🤷‍♂️

In any case, the suggestions here and elsewhere are much appreciated, and I will have time to look into all this in about a week or two.

**Comment by expersso** [comment]
I'm trying to find out why this was actually archived. As mentioned, I got an email about a unit test failing on Solaris, but my patch was accepted and was working on Solaris at the time. Could someone direct me to where I can confirm that it still wasn't working on Solaris even after the patch? I can't seem to find the logs for this.

**Comment by briatte** [comment]
To be honest, I have no idea how to fix/help with this… @fsolt perhaps?

**Comment by fsolt** [comment]
Hmm.  I haven't run into this problem before either.  I see [mention here that rhub.io has a Solaris builder](https://github.com/r-lib/devtools/issues/1393#issuecomment-398830664) and I know devtools now walks you through rhub.io before submitting to CRAN, but that just goes towards confirming what @expersso already said about the patch working on Solaris when he submitted it, nothing more.  I don't see/can't find any way to get to logs of archived packages.

Brainstorming, I guess the two more or less self-evident options are (1) seeking advice from CRAN volunteers (at the risk of wasting their time, plus I suppose of having to wait on a response), or (2) submitting the new revised version and seeing what happens (at the risk of wasting @expersso 's)?
